src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+  Hello, world!
+  </body>
```

Comment:
Content of a tag should also be intended.
```suggestion
  <body>
    Hello, world!
  </body>
```

Reply:
![image](https://user-images.githubusercontent.com/99665973/158166879-dbb82586-bddc-4794-8733-fee8208388dc.png)